zmon-controller-app/src/main/java/org/zalando/zmon/controller/ZMonRestService.java

```diff
@@ -89,7 +91,7 @@ public ZMonRestService() {
     }
 
     @RequestMapping(value = "/updateCheckDefinition")
-    public ResponseEntity<CheckDefinition> updateCheckDefinition(@RequestBody(required = true) CheckDefinitionImport check) throws ZMonException {
+    public ResponseEntity<CheckDefinition> updateCheckDefinition(@RequestBody CheckDefinitionImport check) throws ZMonException {
```

Comment:
Why is `required=true` removed?

Reply:
It is default value, no need to specify.

https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/web/bind/annotation/RequestBody.html